lib/nfc/nfc_device.c

```diff
@@ -627,7 +627,11 @@ bool nfc_device_load_mifare_df_data(FlipperFormat* file, NfcDevice* dev) {
                 *app_head = app;
                 app_head = &app->next;
             }
-            if(!parsed_apps) break;
+            if(!parsed_apps) {
```

Comment:
Could you declare 
#define TAG "NfcDevice"
at the beginning and write logs just 
FURI_LOG_W(TAG, "Non-parsed apps found!");

And please delete commented code